addons/web_editor/views/snippets.xml

```diff
@@ -220,7 +220,7 @@
             />
         </div>
 
-        <div data-js="BackgroundShape" string="Shape"
```

Comment:
@bvr-odoo Was this in fact ignored?

Reply:
@qsm-odoo I dont know why this string attribute is there, its the only one I found on a `div` with a `data-selector` ... It seems useless.. I asked SAD for info
this line is also strange in `_computeSnippetTemplates` (maybe you know more than me)
`'data': _.extend({string: $style.attr('string')}, $style.data()),`

Reply:
I found an other one in V14.0
`<div string="Page URL" data-js="facebookPage" data-selector=".o_facebook_page">`
It also seems useless

Reply:
Strange indeed, the line was introduced with https://github.com/odoo/odoo/commit/72a2f417ae746142a8098129d7629241908769fb#diff-52a4f9d2c217548e69e6b7fd097f286f1754a6389734eea254b87255e501cbefR1330 by me but no idea why as totally unused by my commit it seems. Probably some draft dev that I left. Since you end up using it, let's say I just totally planned it would be useful :smirk: 

But indeed, let's remove the useless addition of string attribute in the XML.